src/libraries/Common/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -22,11 +19,11 @@ internal static partial class ADP
     {
         // NOTE: Initializing a Task in SQL CLR requires the "UNSAFE" permission set (https://docs.microsoft.com/en-us/dotnet/framework/performance/sql-server-programming-and-host-protection-attributes)
         // Therefore we are lazily initializing these Tasks to avoid forcing customers to use the "UNSAFE" set when they are actually using no Async features
-        private static Task<bool> _trueTask;
-        internal static Task<bool> TrueTask => _trueTask ?? (_trueTask = Task.FromResult(true));
+        private static Task<bool>? _trueTask;
+        internal static Task<bool> TrueTask => _trueTask ??= Task.FromResult(true);
```

Comment:
I actually had a hard time resisting a much more general cleanup - there's so much C# goodness that has been added since this code has been written. But these annotation PRs are already huge, so I think it's best for everyone to really stick to annotating. We can always do some cleanup (even automating) after this).